frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/AppearanceSettings.tsx

```diff
@@ -193,7 +207,7 @@ export const AppearanceSettings = ({
             <Text>{jt`This banner appears on all static embeds created with the Metabase open source version. You’ll need to upgrade to ${(
               <ExternalLink
                 key="bannerPlan"
-                href={upgradePageUrl}
+                href={metabaseBannerUpgradeUrl}
```

Comment:
On master the URL was created by an upsell component
https://github.com/metabase/metabase/blob/42dc5239db63f233dd86181f4e7ad0487c5a6d04/frontend/src/metabase/admin/upsells/UpsellMetabaseBanner.tsx#L7-L16

And the URL is
https://www.metabase.com/upgrade/?utm_source=product&utm_medium=upsell&utm_campaign=remove-mb-branding&utm_content=static-embed-settings-look-and-feel&source_plan=oss

I've put the same URL in the test to ensure we use the same URL. (Well not exactly the same as the one here doesn't have a trailing slash after the path, but they're basically the same URL).